docs/app-check/default-providers.md

```diff
@@ -79,12 +79,19 @@ Future<void> main() async {
   await Firebase.initializeApp();
   await FirebaseAppCheck.instance.activate(
     webRecaptchaSiteKey: 'recaptcha-v3-site-key',
-    // Default provider for Android is the Play Integrity provider. You can use the "AndroidProvider" enum to choose
+    // Default provider for Android is the play integrity provider. You can use the "AndroidProvider" enum to choose
```

Comment:
Seems to be uppercase in the website: https://developer.android.com/google/play/integrity

```suggestion
    // Default provider for Android is the Play Integrity provider. You can use the "AndroidProvider" enum to choose
```